roles/confluent.common/defaults/main.yml

```diff
@@ -17,8 +17,8 @@ confluent_common_repository_redhat_dist_enabled: 1
 
 install_java: true
 redhat_java_package_name: java-1.8.0-openjdk
-debian_java_package_name: openjdk-8-jdk
-ubuntu_java_package_name: openjdk-8-jdk
+debian_java_package_name: openjdk-11-jdk-headless
```

Comment:
can you include this in a different PR? does DIGEST-MD5 need java 11?
There may be other considerations we need to make before uping the java version

Reply:
No, of course it's not. Debian 10 isn't required either =)  But, according to docs, java 11 is officially supported.

Anyway, sorry for this, just didn't have much time to split all the small fixes.